src/components/account/AccordionItem.tsx

```diff
@@ -67,10 +67,10 @@ const triggerStyle = css`
 const contentStyle = css`
   overflow: hidden;
   font-size: 0.95rem;
-  background: var(--gray1);
+  background: var(--gray2);
   border-top: 1px solid var(--gray3);
   line-height: 17px;
-  padding: 1rem 1.3rem;
+  padding: 1rem;
 
   &[data-state='open'] {
```

Comment:
&[data-state='closed'] {
    animation: ${slideUp} 0.2s ease-in-out;
    padding-bottom: 0;
  }